utils.py

```diff
@@ -8,4 +8,8 @@ def is_citizen_in_range(target_number: int, citizens: List[Citizen]) -> bool:
 
 
 def validate_citizen_target_number(target_number: int, citizens: List[Citizen]) -> bool:
-    return target_number is not None and is_citizen_in_range(target_number, citizens) and citizens[target_number].is_alive
+    return target_number is not None and is_citizen_in_range(target_number-1, citizens) and citizens[target_number-1].is_alive
+
+
+def stage_validate_citizen_target_number(target_number: int, citizens: List[Citizen], creator: Citizen) -> bool:
+    return target_number is not None and is_citizen_in_range(target_number-1, citizens) and citizens[target_number-1] is not creator and citizens[target_number-1].is_alive
```

Comment:
This code won't be used outside of StageEffect, we don't need to move this logic to utils module (which contain commonly used code).

If you want to avoide code duplication in StageEffect - you can write:
return (utils.validate_citizen_target_number(target_number, citizens) and citizens[target_number-1] is not creator)
inside validate method of StageEffect

Reply:
Done